tests/testthat/test-applicants_regression.R

```diff
@@ -0,0 +1,10 @@
+context("applicants regression")
+max_year <- 2015
+
+test_that(paste0("applicants head and tail between start and ", max_year, " unchanged"), {
```

Comment:
I don't think you need such a complex test name.

Reply:
And I think it might make more sense to put all the tests in a single file, and call it `test-data.R`

Reply:
Also you shouldn't be filtering the max year; we want to see when something changes in the tail of the data.